### PR TITLE
fix(tokens): emit valid JSON from /nf:tokens --json on empty data (edge-path F47)

### PR DESCRIPTION
**Pull request by jobordu**
## What

`/nf:tokens --json` printed the human string `No token usage data found.` (non-JSON) on a fresh project with no usage data. `token-dashboard.cjs`'s empty-records guard returned **before** the `--json` branch, so any caller doing `JSON.parse($(node bin/token-dashboard.cjs --json))` would throw.

## Fix

The empty path now returns the same empty-shape object as the populated `--json` result:
```json
{ "slots": {}, "sessions": {}, "total": { "input": 0, "output": 0, "estimatedCost": 0 } }
```
The human (non-`--json`) path is unchanged — still prints `No token usage data found.`

## How it was found

First finding of an **error/edge-path dogfooding round**: a probe runs every skill-captured `--json` script under missing/corrupt state and flags non-JSON-under-`--json`, crashes, and hangs. This is the same class as the campaign's F28 (`check-provider-health --json`).

## Test

New behavioral assertion in `bin/token-dashboard.test.cjs` — proven **red without the fix, green with it**. Full file 18/18 green.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Bug Fixes
* Fixed `/nf:tokens --json` command to return valid JSON with a consistent structure when no token usage data is available, replacing the previous human-readable message

## Tests
* Added test coverage to ensure JSON output remains properly formatted for edge cases with missing or empty token usage data

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Format follows [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
...
 
 ### Fixed
+- fix(tokens): `/nf:tokens --json` now emits valid JSON when there is no usage data (dogfooding edge-path F47). `token-dashboard.cjs`'s empty-records guard returned the human string `No token usage data found.` *before* the `--json` branch, so `--json` on a fresh project (no `.planning/telemetry/token-usage.jsonl`) printed non-JSON — any caller doing `JSON.parse($(… --json))` would throw. The empty path now returns the same empty-shape object as the populated result (`{slots:{},sessions:{},total:{input:0,output:0,estimatedCost:0}}`); the human (non-`--json`) path is unchanged. Found by an error-path probe that runs every skill-captured `--json` script under missing/corrupt state. New behavioral assertion in `bin/token-dashboard.test.cjs`.
```
[the remark this answers is not in the document]
Comment (jobordu) [reply CHANGELOG.md]: Good catch — fixed in the follow-up commit: the example now uses quoted keys (`{"slots":{},...}`), valid JSON matching the actual output.
